src/cmd/flux-job.c

```diff
@@ -632,13 +642,31 @@ int cmd_id (optparse_t *p, int argc, char **argv)
     return 0;
 }
 
+enum {
+    CONTEXT_FORMAT_TEXT = 0,
+    CONTEXT_FORMAT_JSON = 1,
+};
+
 struct eventlog_ctx {
     struct flux_kvs_eventlog *log;
     optparse_t *p;
     flux_jobid_t id;
+    int format;
 };
 
-void output_event (const char *event)
+int context_format (optparse_t *p)
```

Comment:
I don't think converting to an enum is really buying us anything over a strcmp on the original option value.

Reply:
I think I was overthinking future support, where there would be other future output formats.  It is a bit overkill.